30412/Mihut Alina/assignment 2/assignment 2.1/dll.h

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Alina Mihut on 3/7/20.
+//
+
+#ifndef ASSIGNMENT_2_1_DLL_H
+#define ASSIGNMENT_2_1_DLL_H
+#include <stdio.h>
+#include <stdlib.h>
+typedef struct node {
```

Comment:
Aici - practic trebuia să mai ai o structură care să conțină lista ta - te poți uita în laborator sau în exemplul de Snake pe care l-am scris eu